### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by rhagarty**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # Create a calorie counter mobile app using Watson Visual Recognition
 
-In this developer journey, we will create a calorie counter mobile app using Apache Cordova, Node.JS and Watson Visual Recognition.This mobile application analyzes the captured image(food) using Watson Visual Recognition and extracts nutritional information.
+In this developer journey, we will create a calorie counter mobile app using Apache Cordova, Node.js and Watson Visual Recognition. This mobile app extracts nutritional information from captured images of food items. 
 
-![Architecture Diagram](https://github.com/RiyaMRoy04/CalorieCounterApp/blob/master/images/arch_diagram.png "Architecture_diagram")
+Currently this mobile app only runs on Android, but can be easily ported to iOS.
+
+![](https://github.com/IBM/CalorieCounterApp/blob/master/images/arch_diagram.png)
```
Comment (stevemart) [inline README.md]: using just `` would work, also, it should be `doc/architecture.png`

File: README.md [hunk 2]
```diff
@@ -18,103 +18,187 @@ Want to take your Watson app to the next level? Looking to leverage Watson Brand
...
 
-If you haven't so yet, you also need to download and install Node.js and cordova module. Follow steps below to install
- 
- 1. INSTALL NODE.JS- Download and install [Node.js](https://nodejs.org/en/download/). On installation you should be able to invoke node and npm on your command line.
- 
- 2. INSTALL CORDOVA CLI- Install the cordova module using npm utility of Node.js. The cordova module will automatically be downloaded by the npm utility. Follow the steps [here](https://cordova.apache.org/docs/en/latest/guide/cli/#installing-the-cordova-cli) to install the corodva module. Check if you satisfy requirements for building the platform [here](https://cordova.apache.org/docs/en/latest/guide/cli/#install-pre-requisites-for-building)
- 
- 3. INSTALL REQUIREMENTS FOR BUILDING CORDOVA APP- Few requirements(JDK,Android SDK) have to be satisfied to build the cordova project. Cordova for Android requires the Android SDK and requirements can be installed from [here](https://cordova.apache.org/docs/en/latest/guide/platforms/android/index.html#installing-the-requirements)
+![](doc/source/images/toolchain-pipeline.png)
```
Comment (stevemart) [inline README.md]: see above comment

File: README.md [hunk 3]
```diff
@@ -18,103 +18,187 @@ Want to take your Watson app to the next level? Looking to leverage Watson Brand
...
+export PATH=${PATH}:/users/joe/Android/sdk/platform-tools:/users/joe/Android/sdk/tools
+```
+
+# License
   
+[Apache 2.0](LICENSE)
+
+# Privacy Notice
+
+If using the Deploy to Bluemix button some metrics are tracked, the following
```
Comment (stevemart) [inline README.md]: this section is only applicable if we're using the d2b deployment tracker, which i don't think we are, unless you plan to do that, remove it

Comment (RiyaMRoy04) [reply README.md]: @stevemart d2b deployment tracker is required since we are pushing server code to Bluemix

**Pull request merged**